src/main/java/io/jenkins/plugins/file_parameters/StashedFileParameterDefinition.java

```diff
@@ -54,6 +54,11 @@
             return "Stashed File Parameter";
         }
 
+        @Override public String getHelpFile()
```

Comment:
You should never need to override `getHelpFile` except when you want to share one file among multiple descriptors. Instead, simply create `help.html` in the same directory as `config.jelly`.

Reply:
there is no config.jelly for this as it extends AbstractFileParameterDefinition

Reply:
You put it in the directory `config.jelly` would have been in: `src/main/resources/io/jenkins/plugins/file_parameters/StashedFileParameterDefinition/`

Reply:
🤦‍♂️